packages/visual-regression-tests/action.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Chromatic Deployment'
+
+# Event for the workflow
+on: push
+      branches: 
+            - sb/**
```

Comment:
based on: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags